cli/src/pcluster/schemas/cluster_schema.py

```diff
@@ -752,6 +751,7 @@ class SlurmQueueNetworkingSchema(QueueNetworkingSchema):
         PlacementGroupSchema, metadata={"update_policy": UpdatePolicy.MANAGED_PLACEMENT_GROUP}
     )
     proxy = fields.Nested(QueueProxySchema, metadata={"update_policy": UpdatePolicy.QUEUE_UPDATE_STRATEGY})
+    enable_single_availability_zone = fields.Bool(metadata={"update_policy": UpdatePolicy.QUEUE_UPDATE_STRATEGY})
```

Comment:
Did we test the use of keeping this `UpdatePolicy.QUEUE_UPDATE_STRATEGY` for this paramter?